versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,104 @@
+
+1. Enter the **Hostname** for Rancher. The Rancher hostname must be a fully qualified domain name (FQDN). The Rancher server URL will be created using this hostname.
+   :::note
+
+   The IP address of the Rancher hostname must be resolvable by a public DNS.
+
+   :::
+1. Using the slider, select the number of **Replicas**.
+1. Choose and confirm a **Bootstrap Password**. During the first login, you will use the bootstrap password to authenticate to the Rancher dashboard.
+   :::note
+
+   The current Rancher deployment exposes the bootstrap password in the Cluster configuration settings in the Azure Portal. Until this security issue is resolved, we suggest changing the Admin password after initial login, by editing your profile in the Rancher dashboard.
+
+   :::
+1. Select **Next**.
+
+### Review + Create
+
+1. On the **Review + create** tab, review the summary of the offer (Price, Basics, Rancher Configuration) and the link to **view automation template** (Azure Resource Manager Template).
+1. Select **Create** to start the deployment.
+
+### Deployment Complete
+
+When the deployment successfully completes, Rancher Prime will be installed.
+
+:::note
+
+On the **Extensions + applications** page, the **Provisioning State** may show **Succeeded** even though the deployment may still be in progress. You can monitor the deployment progress by logging into the AKS cluster and looking at the **rancher-cloud** deployment.
+
+:::
+
+## Log into the Rancher Dashboard
+
+You may now log in to the Rancher dashboard by pointing your browser to the Rancher server URL `https://<host-name>`. The `<host-name>` is the hostname you entered when you [installed Rancher](#installing-rancher).
+
+:::note
+
+The Rancher hostname must be resolvable by public DNS. See the [Prerequisites](prerequisites.md) for more details.
+
+:::
+
+## How to Use Rancher
+
+After you login to Rancher Prime, you should notice the **Welcome to Rancher Prime** message at the top of the screen.
+
+![Rancher Prime Home](/img/install-rancher-prime-home.png)
+
+If your Rancher Prime PAYG deployment only has **Welcome to Rancher** at the top of the screen, make sure that you've updated to the latest version, and reset the branding to default (i.e., "suse") from **Global Settings**.
+
+![Global Settings](/img/install-rancher-prime-global-settings.png)
+
+## Rancher Prime PAYG Billing
+
+View billing information in the Azure Portal by going to **Home** > **Cost Management (subscription) | Cost analysis**.
+
+## Uninstalling Rancher Prime PAYG Offering
+
+Uninstalling Rancher Prime may not remove all of the Kubernetes resources created by Rancher. Run the [Rancher resource cleanup script](https://github.com/rancher/rancher-cleanup) to perform a more comprehensive cleanup.
+
+The best practice for uninstalling the Rancher Prime PAYG offering is to migrate any non-Rancher workloads to a different cluster and destroy the Rancher cluster.
+
+:::note warning
```

Comment:
```suggestion
:::warning
```

Warning is built-in so it doesn't need to use a custom label.

https://docusaurus.io/docs/markdown-features/admonitions